src/test/java/org/symphonyoss/symphony/messageml/elements/ElementTest.java

```diff
@@ -293,4 +294,10 @@ public void testBodyInvalidParent() {
     }
   }
 
+  protected void assertSameBiItem(BiItem expected, BiItem actual) {
+    // assertEquals(actual, expected);
```

Comment:
to remove?

Reply:
@symphony-mariacristina added "equals" method for BIItem in her PR, this commented line will be used instead of the two lines below when the PR is merged.